Tests/Schema.NET.Test/core/BookTest.cs

```diff
@@ -108,7 +108,7 @@ public class BookTest
                         "}," +
                         "\"expectsAcceptanceOf\":{" +
                             "\"@type\":\"Offer\"," +
-                            "\"availability\":\"https://schema.org/InStock\"," +
+                            "\"availability\":\"schema:InStock\"," +

```

Comment:
Can you please explain the URL change here? Were we incorrectly handling enumeration types?

Reply:
It seems that Schema.org changed enumeration values to be `schema:EnumerationName`. I updated our JSON converter to read the old HTTP/HTTPS versions as well as this new version. It writes the new version as that is what the JSON files from Schema.org provide. The test though does a exact string match so it needed to be updated to match the new value.

To be honest, I don't like that Schema.org changed it as I imagine it is a breaking change for anyone who stored the value. I think though we should use whatever value they are currently providing though.

Reply:
Have you seen a doc or GitHub issue discussing this change? It looks really strange that they'd go and break everyone like this. If we want to be interoperable with others, consumers of this package will want to output the full URL's instead of the new format.

Reply:
I think I worked out why the change has happened. In prior versions of the JSONLD file we download, they had the full URL. In the new one, they don't BUT they have something else added to the top most context:

![image](https://user-images.githubusercontent.com/904226/109585663-d5899e00-7b53-11eb-8234-646def9932ba.png)

The "schema" property on "@context" has the URL - I think the intention is to replace "schema:" with "https://schema.org/". Will fix that up and revert those "schema:" specific changes I've made.